### PR TITLE
[EEX-773] consensus api move on echorand api

### PR DESCRIPTION
**Pull request by AlinaLoz**

### CODE REVIEW

**Review by chralsia — DISMISSED** [review]

File: docs/API.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ try {
 <dd></dd>
 <dt><a href="#getProposedTransactions">getProposedTransactions(accountNameOrId)</a> ⇒ <code>Promise.&lt;*&gt;</code></dt>
 <dd></dd>
-<dt><a href="#getContractLogs">getContractLogs(contractId, topics, fromBlock, toBlock)</a> ⇒ <code>Promise.&lt;Array.&lt;ContractLogs&gt;&gt;</code></dt>
+<dt><a href="#getContractLogs">getContractLogs(cb, contractId, topics, fromBlock, toBlock)</a> ⇒ <code>Promise.&lt;Array.&lt;ContractLogs&gt;&gt;</code></dt>
```
Comment (vkrasutsky) [inline docs/API.md]: we need to remove cb

File: .gitignore [hunk 2]
```diff
@@ -70,3 +70,4 @@ apidoc
```
Comment (vkrasutsky) [inline .gitignore:70]: build is failed

File: types/echo/api.d.ts [hunk 3]
```diff
@@ -79,12 +79,12 @@ export default class Api {
 	getContractPoolWhitelist(contractId: string): Promise<unknown>;
 	getContractHistory(operationId: string, stop: number, limit: number, start: number): Promise<Array<ContractHistory>>;
 	getContracts(contractIds: Array<string>, force?: boolean): Promise<Array<{id: string, statistics: string, suicided: boolean}>>;
-	getContractLogs(opts: {
+	getContractLogs(cb: () => any, opts: {
```
Comment (vkrasutsky) [inline types/echo/api.d.ts]: rm cb

File: src/echo/api.js [hunk 4]
```diff
@@ -430,6 +430,8 @@ import { PublicKey } from '../crypto';
 *  		address:String,
 *  		log:Array.<String>,
 *  		data:String
+ *  	trx_num:Number,
```
Comment (vkrasutsky) [inline src/echo/api.js:433]: indent

File: .travis.yml [hunk 5]
```diff
@@ -13,6 +13,9 @@ jobs:
       if: tag IS blank
       name: "Test lint and audit"
       script:
+        - rm -rf node_modules && rm package-lock.json
```
Comment (vkrasutsky) [inline .travis.yml]: ?

File: src/echo/subscriber.js [hunk 6]
```diff
@@ -548,7 +548,7 @@ class Subscriber extends EventEmitter {
 	 *  @return {Promise.<undefined>}
 	 */
 	async _setConsensusMessageCallback() {
-		await this._wsApi.networkNode.setConsensusMessageCallback(this._echorandUpdate.bind(this));
+		await this._wsApi.echorand.setConsensusMessageCallback(this._echorandUpdate.bind(this));
```
Comment (vkrasutsky) [inline src/echo/subscriber.js]: rename setConsensusMessageCallback

File: src/echo/ws-api/echorand-api.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+	}
+
+	/**
+     *  @method setConsensusMessageCallback
+     *
+     *  @param  {Function} callback
+     *
+     *  @return {Promise.<null>}
+     */
+	setConsensusMessageCallback(callback) {
```
Comment (vkrasutsky) [inline src/echo/ws-api/echorand-api.js]: rename

File: src/echo/ws-api/wallet-api.js [hunk 8]
```diff
@@ -447,6 +436,7 @@ class WalletAPI {
 			string.toRaw(brainKey),
 			string.toRaw(accountName),
 			string.toRaw(accountNameOrId),
+			evmAddress,
```
Comment (vkrasutsky) [inline src/echo/ws-api/wallet-api.js:439]: toRaw?

**Review by vkrasutsky — APPROVED** [review]

**Pull request merged**